include/s3select_functions.h

```diff
@@ -85,6 +86,7 @@ class s3select_functions
     {"nullif", s3select_func_En_t::NULLIF},
     {"between", s3select_func_En_t::BETWEEN},
     {"isnull", s3select_func_En_t::IS_NULL},
+    {"in", s3select_func_En_t::IN},
```

Comment:
review 
/void push_in_predicate::operator()/
what is the operator name?

Reply:
Thanks. Will update that